### PR TITLE
Remove html coverage and coverage of request files

### PR DESCRIPTION
**Pull request by BrainMaestro**

### CODE REVIEW

**Comment by BrainMaestro** [comment]
Approved

**Comment by SkaeX** [comment]
Approved.

File: phpunit.xml [hunk 1]
```diff
@@ -20,6 +20,7 @@
                 <directory suffix=".php">app/Providers</directory>
                 <directory suffix=".php">app/Console</directory>
                 <directory suffix=".php">app/Exceptions</directory>
+                <directory suffix=".php">app/Http/Requests</directory>
```
Comment (tenorz007) [inline phpunit.xml]: Why remove requests? @BrainMaestro

Comment (BrainMaestro) [inline phpunit.xml]: You can't test them

Comment (tenorz007) [inline phpunit.xml]: Functional testing fa

**Comment by BrainMaestro** [comment]
True. Forgot about that.

**Pull request merged**